source/model/Device.py

```diff
@@ -0,0 +1,39 @@
+from abc import ABC, abstractmethod
+
+
+class Device(ABC):
+    cntr = 0
+
+    def __init__(self):
+        self.deviceId = self.__class__.newId()
+        self._activators = list()
+
+    @classmethod
+    def newId(cls):
+        cls.cntr += 1
+        return cls.cntr
+
+    @property
+    @abstractmethod
+    def name(self):
+        """ The name. So that we might identify different type of devices """
```

Comment:
With this we mean that a each plugin has its own name. 
For example, Philips_lock, Klik_aan_klik_uit_slot. They are all of the type lock, but are different implementation.